### ISSUE TITLE
Can't get reference from canvas out and into the GAN code to put the image in the right place

### ISSUE THREAD

**Issue by kylemath** [kylemath/faceoff#3]
Working in https://github.com/kylemath/lags-bs/pull/2

I've got a canvas
and I've got a gan making an image

I want to write the image to the canvas

Both are drawn in the bottom of PageSwitcher.js,

```
<Canvas draw={draw}>
``` 
- this loads this canvas wrapped in a react component, take props draw, to tell it what to draw, 
Inside this component a reference to the canvas canvasRef is made, 

and returned:
```  
return <canvas ref={canvasRef} {...rest}/>
```

SO

Now I am back in pageSwitcher.js, and at the bottom when the mock data button is pressed, the GAN is setup and generates and image

```
model_runner.generate();
```

model_runner is an instance of the ModelRunner Class imported from GAN.js

its function generate()
calls a another function in GAN.js called computing_generate_main() - which is the code I showed you earlier today where the latent vector is made

this function ends in 
```
await tf.toPixels(y, c); 
```

where c is the canvas reference to print the pixels of the generated image into

SOO @korymath @keyfer  - how do I get the canvasRef which is returned by the Canvas react component, and pass it into the model_runner.generate() code, so it can pass it through to the right place?

tldr: if this is how I print the canvas, how do I get its reference back out ? 

```
 <Card sectioned>
        <Canvas draw={draw} />
      </Card>
```
